src/utils/utils.js

```diff
@@ -400,12 +401,18 @@ export const getStartStopPatch = (vm, start) => {
 
 export const getResource = (
   model,
-  { name, namespaced = true, namespace, isList = true, matchLabels, matchExpressions } = {
+  { name, namespaced = true, namespace, isList = true, matchLabels, matchExpressions, prop } = {
     namespaced: true,
     isList: true,
   }
 ) => {
-  const res = { kind: model.kind, namespaced, namespace, isList, prop: model.kind };
+  const res = {
+    kind: model === NamespaceModel ? ProjectModel.kind : model.kind,
```

Comment:
non-admin user cannot list all namespaces (it wont return only namespaces available to user but 403 forbidden). Instead we need to use ProjectModel which will return available projects/namespaces

Reply:
Sure, comment added to code